packages/web-ui/src/ds/molecules/Chat/MessageList/index.tsx

```diff
@@ -16,21 +17,20 @@ export function MessageList({
   separator?: boolean
 }) {
   return messages.map((message, index) => (
-    <>
```

Comment:
Open the browser in a bigger screen and check Dev tools 🙏 
Otherwise all the warnings like missing key in a loop pass unnoticed